content/synchronization_model.tex

```diff
@@ -19,9 +19,9 @@
 \begin{tabular}{p{0.2\textwidth} | p{0.7\textwidth}}
 {}
 &
-{ Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
+Waits for a symmetric variable to be updated by a remote \ac{PE}. Should be
```

Comment:
Yes, I agree, that is reflected in the changes to shmem_wait. This text here is in reference only to the diagram.